test/unit/arch/new_arch_first_tests.cpp

```diff
@@ -0,0 +1,198 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+**/
+//==================================================================================================
+#include "test.hpp"
+
+#include <eve/arch/wide.hpp>
+
+// For a new arch.
+//
+// After getting meta things to work, this is the first set of tests that
+// need to pass.
+//
+// Comment out 1 by 1.
+
+TTS_CASE("Implement basic make function for wide<int>")
+{
+  eve::wide<int> x {[](int i, int) { return i; }};
+
+  (void)x;
+  TTS_PASS();
```

Comment:
can't this test be merged with the next one ?

Reply:
I duplicate tests to reduce on horrible horrible compiler messages first.
All of these are unnessary  - we have this stuff tested already